lib/Compose.php

```diff
@@ -745,7 +745,7 @@ public function buildAndSendMessage(
         $body, $header, IMP_Prefs_Identity $identity, array $opts = array()
     )
     {
-        global $injector, $prefs, $registry, $session;
+        global $injector, $prefs, $registry, $session, $conf;
```

Comment:
Adding yet another implicit global is not ideal but out of scope for this patch.
Refactoring this to dependency injection via the Core/Config object is an exercise for later.

Reply:
I had a 'yuck' moment at seeing how other functions in `Compose` handled this, but didn't find an easy way to do it properly, hence adopting the global.